src/classes/issues-processor.ts

```diff
@@ -268,6 +317,130 @@ export class IssuesProcessor {
       );
     }
 
+    if (this.options.mlflow) {
+      const {startDate, daysBeforeReplyReminder, daysBeforeAssigneeReminder} =
+        this.options;
+      if (
+        issue.milestone ||
+        issue.isStale ||
+        (startDate &&
+          !isDateMoreRecentThan(
+            new Date(issue.created_at),
+            new Date(startDate)
+          ))
+      ) {
+        return;
+      }
+
+      const reminderToMaintainers = 'Reminder to MLflow maintainers';
+      if (issue.isPullRequest) {
+        // TODO
+      } else {
+        const hasClosingPr = issue.labels.some(
+          ({name}) => name === 'has-closing-pr'
+        );
+        if (hasClosingPr) {
+          issueLogger.info('This issue has a closing PR');
+          return;
+        }
```

Comment:
If the issue has a closing PR, we should still triage it. Else, we might miss the issue / PR.

Reply:
so we still want to post reminders to the issue?